src/software/thunderscope/replay/proto_logger.py

```diff
@@ -5,10 +5,10 @@
 import os
 import logging
 import gzip
-import proto
 from proto.import_all_protos import *
 from extlibs.er_force_sim.src.protobuf.world_pb2 import *
 from software.thunderscope.replay.replay_constants import *
+from typing import Callable
 
 
 class ProtoLogger(object):
```

Comment:
Unrelated, but since we're cleaning up code might as well:
```suggestion
class ProtoLogger:
```
Subclassing object is useless in python3